### PR TITLE
Linting and cleanup

### PR DESCRIPTION
**Pull request by ColeDCrawford**
Mostly linting config. Nginx and prod Dockerfile still WIP.

### CODE REVIEW

**Comment by ColeDCrawford** [comment]
Added an override Compose file to build the prod image. `Dockerfile` works with `nginx` and `gunicorn`.

File: dle/Dockerfile.tests [hunk 1]
```diff
@@ -5,63 +5,27 @@ WORKDIR /app
...
+RUN curl -L https://github.com/mozilla/geckodriver/releases/download/v0.32.2/geckodriver-v0.32.2-linux64.tar.gz | tar xz -C /usr/local/bin
+RUN chmod +x /usr/bin/firefox
+# set display port to avoid Selenium crash
 ENV DISPLAY=:99
 
 # Python requirements
 COPY requirements.txt /app
-RUN pip install -r requirements.txt
+COPY requirements-dev.txt /app
+RUN pip install -r requirements-dev.txt
```
Comment (bcavezza-NGT) [inline dle/Dockerfile.tests:23]: Do we also install `requirements.txt` in order to run the tests in Docker?

**Review by bcavezza-NGT — APPROVED** [review]

File: dle/Dockerfile.tests (hunk 1 above)
Comment (ColeDCrawford) [reply dle/Dockerfile.tests:23]: `requirements-dev` calls `requirements` in the first line: `-r requirements.txt`. So they do get installed.

Comment (ColeDCrawford) [reply dle/Dockerfile.tests:23]: That's fairly typical, it does mean that the linting CI setup takes longer. We need everything for the Django tests, but only a couple packages for linting

**Review by agiannini — APPROVED** [review]
looks good

**Pull request merged**